imagetest/test_suites/packagevalidation/package_test.go

```diff
@@ -38,41 +61,74 @@ func TestStandardPrograms(t *testing.T) {
 func TestGuestPackages(t *testing.T) {
 	utils.LinuxOnly(t)
```

Comment:
Doesn't need to happen in this PR but we should (at some point) expand this to include windows packages from `googet installed`.

Reply:
Yeah that'll be coming in a future PR